---
title: Throwing a net around the sea
description: Environmental DNA has been acclaimed as a transformative resource for
  understanding marine biodiversity — but who gets to draw from this rich well of
  ecological data?
subtitle: Environmental DNA has been acclaimed as a transformative resource for understanding
  marine biodiversity — but who gets to draw from this rich well of ecological data?
authors:
- c3630266-0b29-4a80-8b04-f2a63c9b5d6d
image:
  path: public/tn0zn0oebgwygrie0cxmofzqlfh2/coverimage.jpg
  description: ''
show_biography: false
locales: []
draft: false
order: 13
layout: post
uuid: c97be219-8d2c-4106-9cb8-4762561c47be
liquid: false
usuaries:
- 644
- 726
last_modified_at: 2023-05-29 18:26:59.967954311 +00:00
---

<p style="text-align:start">Collecting a marine environmental DNA sample is simple.</p><p style="text-align:start">First, prepare yourself for sample collection. Wear gloves — as sanitized as you can make them — to prevent contamination. The goal is to collect DNA from the myriad species that have passed through your chosen site, a tiny window of the vast sea; any residual material derived from your own skin, or from that of your cat, would only make this miraculous task more difficult. Wear waterproof boots, too — less for the sake of the data and more for your own comfort as you step through the coastal landscape.</p><p style="text-align:start">Next, find your sampling location. Within the broader site you survey, there may be a significant degree of ecological variation — between a tide pool and the more open ocean, or even from two different patches of ocean at different depths. The time of year in which you collect your samples also matters. Species move in and out of given areas throughout the year, so their presence in eDNA samples fluctuates as they come and go. It’s best, if you can, to sample multiple times across multiple subsites. Bring friends if you wish, whether for increased efficiency or companionship. You’re helping out with a grand scientific project — there’s no need to be lonely.&nbsp;</p><p style="text-align:start">At this time, you’ll need to actually collect your sample. There are a few options with which to perform this task. In a pinch, any sufficiently sanitized, sealable container can be used. Most opt to use one liter Nalgene bottles — those standard plastic lab and field research workhorses. For better-resourced users there exist specialized tools, ranging up to the $7,695 eDNA Sampler Backpack Combo, a twenty-four -pound luxury offering available from fishery science supplier Smith-Root that not only collects samples but filters them too<a href="#fn1" id="fnef1" role="doc-noteref" data-turbo="false"><sup>1</sup></a>.</p><p style="text-align:start">Regardless of the amount you spend on your tools, the core act of collection is dead simple. Just move whatever implement you have through the body of water you seek to sample. In the moment, it may feel as if you have been returned to a childhood trip to the beach, the play-act of swishing a bucket around in the ocean turned into a scientific procedure. But this moment of nostalgic conjuring cannot last forever; your work doesn’t end here.</p><p style="text-align:start">If you don’t have $8000 handy, filtration is still feasible — and necessary. In this liter of sea you have collected, there may be the traces of untold thousands of species. The only way to find out which traces remain is by filtering your sample, separating the genetic wheat from the aquatic chaff.&nbsp;</p><p style="text-align:start">You can use a number of methods with various degrees of difficulty at this stage. Perhaps the most common involves using a vacuum pump to move the water through a 0.2 μm or 0.45 μm cellulose nitrate filter, though a new, lower-tech strategy removes the pump entirely, and allows gravity to pass the water from suspended sample bags down an IV line through the filters<a href="#fn2" id="fnef2" role="doc-noteref" data-turbo="false"><sup>2</sup></a>. Regardless of how you get there, the end result is the same — the molecular composition and size of the filters catch DNA fragments within their mesh, removing them from the original context of the water you sampled and digesting them into a format more easily analyzed.</p><p style="text-align:start">Now, after whatever magic you worked to filter your sample down, you’ll have to extract and process the DNA contained within. If you’re new to the world of eDNA, it’s unlikely that you will be processing these samples yourself. If you’re lucky, you’ll be working with a local academic lab with their very own next-generation sequencing platform — an extremely expensive box that transmutes DNA into data. More plausibly, you may have to send your samples (stored at low, but not freezing, temperatures in order to preserve the DNA) to one of a relatively limited number of centralized commercial labs that will process them for you.&nbsp;</p><p style="text-align:start">Then, you wait.</p><p style="text-align:start">How long you wait depends on what you’re looking for. If you know exactly which species you want to detect, then you’ve got a faster path to results. The quantitative real-time Polymerase Chain Reaction (qtPCR) — a similar method to the one used in many COVID-19 tests — can provide results in fewer than twenty-four hours, depending on the precise methods used. QtPCR is a finely-pointed tool, capable of detecting the presence of a single specified species by matching sample DNA to a species-specific “barcode”, a primer, or small strands of prepared DNA, that only binds to sample DNA of a particular species. If you want to instead understand the broader ecological composition of your site, you’re going to want to use eDNA metabarcoding, a complimentary but more time-consuming tactic. A metabarcoding assay takes a broader view; instead of using just one barcode it uses a “library” of them, each corresponding to a different species. That wider consideration allows for a metabarcoding assay to give you a rough summary of the species that passed through your sampling location — an entire world of life, distilled down into data.&nbsp;</p><p style="text-align:start">But even after the sample is processed, extracted, and analyzed, it can take months or even years for the results of your study to reach you. Academic publishing is no one’s idea of an efficient process, and scientific journals are not designed to get results to the general public in a speedy fashion. If you’re lucky, you’ll be able to find the results on some repository months from now. If you’re less lucky, your contribution to the process of science might end up hidden behind a paywall, or minimized down to a statement that “data will be made available upon request”.</p><p style="text-align:start">For now, though, you wait.</p><h3 style="text-align:start" id="knowledge-hunters-knowledge-gatherers">Knowledge Hunters, Knowledge Gatherers</h3><p style="text-align:start">What <em>are </em>you waiting for, anyways?</p><p style="text-align:start">Scientific journals and trade publications claim that environmental DNA metabarcoding is a transformative tool for studying biodiversity. It’s a method that takes the fruits of the genomic revolution — the advent of a set of techniques known collectively as “next-generation sequencing,” which brought astonishing increases in the speed at which DNA could be sequenced and analyzed around the turn of the new millennium — and trains their focus on one of the most deceptively difficult questions in ecology: What species live in a given habitat?</p><p style="text-align:start">We have been collecting incomplete answers to this question even before ecology was defined as a discipline. For generations, humans have observed the other living things around them, and attempted to make sense of how and where they lived. At its most practical, this enterprise took the form of fishers, hunters, and gatherers sussing out the best spots to collect their particular quarries. At its most extravagant, it expressed itself in the lavish menageries of ancient monarchs like Nebuchadnezzar and Alexander the Great, with curiosities from the furthest corners of their empires collected as living symbols of the vastness of their dominion. These menageries over time evolved into the modern zoo, reborn in early modern Europe and Victorian Britain as places for the public to observe the scientific fruits of imperialism and empiricism.</p><p style="text-align:start">From herb-gatherers to emperors, informal seekers of ecological knowledge throughout history all desired to possess their objects of study. To them, it was not enough to merely know that a given species was present in a particular locale, but that the species could be captured and mastered. For most of their existences, ecology and zoology have played by the same rules — witness Harvard University’s Museum of Comparative Zoology, a slightly-macabre monument to a century and a half of animal artifact collection<a href="#fn3" id="fnef3" role="doc-noteref" data-turbo="false"><sup>3</sup></a>. Even early genetic techniques in the study of biodiversity required researchers to engage physically with their subjects of study, either living or dead — the cost and difficulty of early genetic sequencing meant that using anything but the most well-preserved samples was a fool’s game. These pursuits of knowledge-gathering, with their undercurrent of physical domination, still exist in the shadow of what historian Thomas Richards refers to as “The Imperial Archive,” Victorian Britain’s drive to collect data and physical resources from all corners of its colonial dominion in order to understand and govern it<a href="#fn4" id="fnef4" role="doc-noteref" data-turbo="false"><sup>4</sup></a>.</p><p style="text-align:start">Environmental DNA offers a break from the need to capture — or even encounter — specimens. More than just a method for reducing labor in the field, eDNA promises that researchers will no longer have to invasively extract DNA from physical bodies at all. Instead, eDNA researchers can let natural processes of degradation do the work for them, exploiting the past two decades’ increases in sensitivity and significant decreases in cost of sequencing in order to collect detailed taxonomic information about what lives where — without having to actually see any of them in person. While the bodies of scientific papers using eDNA themselves tend to remain in the dry register of contemporary scientific writing, subtle hints of mystic awe seep through in some of the more public-facing writing around eDNA. This awe is expressed in scientific press releases claiming that eDNA can capture the “Ocean in a Cup” and papers quoting Thomas Huxley’s (possibly apocryphal) saying that “a drop of water is as powerful as a thunderbolt” <a href="#fn5" id="fnef5" role="doc-noteref" data-turbo="false"><sup>5</sup></a><a href="#fn6" id="fnef6" role="doc-noteref" data-turbo="false"><sup>6</sup></a>. At its most powerful, eDNA sampling feels not just like a step forward in the slow walk of methodological improvements but also a leap into some new scientific future, capable of results that would have seemed miraculous prior to its advent.</p><h3 style="text-align:start" id="residence-times">Residence Times</h3><p style="text-align:start">In sufficiently cold soil, eDNA maintains itself indefinitely. Last fall, researchers from the University of Copenhagen announced that they had retrieved a 2-million-year-old environmental DNA sample from the long-frozen soil of the northernmost reaches of Greenland<a href="#fn7" id="fnef7" role="doc-noteref" data-turbo="false"><sup>7</sup></a>. When they sequenced it, they found a rare window into the recesses of deep time — a genetic roster of the ancient Arctic landscape replete with mastodons, reindeer, geese, and horseshoe crabs.&nbsp;</p><p style="text-align:start">In water, though, DNA is not as lucky. In a stream or sea, DNA degrades into meaninglessness in a matter of days<a href="#fn8" id="fnef8" role="doc-noteref" data-turbo="false"><sup>8</sup></a>. If soil creates palimpsests out of eDNA, different scenes layered upon each other to build an obscured archive, then water is an etch-a-sketch, its contents constantly in flux without continuity from moment to moment.</p><p style="text-align:start">Water has certain advantages over soil for prospective eDNA miners, though. Chief among them is that water is a<em> </em>solvent — within it, DNA mixes homogeneously into solution, creating an uniformly DNA-laden cocktail that can be sampled with less fuss than a similar volume of soil, where DNA may distribute itself less evenly. Water’s penchant for degrading DNA can even help with certain applications of eDNA studies. Due to the shorter residence time of DNA in water, sampling the same area of coast or lake over time can create a portrait of changing conditions — a feature that is of interest to researchers studying invasive species introductions or climate migrations. While collecting each individual bag or bottle of seawater collected doesn’t feel like much, in aggregate, drawing those liters of data out of the sea allows a scientist gradually to form a picture of the whole of an ecosystem in both space and time.</p><h3 style="text-align:start" id="community-ecology">Community Ecology</h3><p style="text-align:start">Marine eDNA studies have quickly become a much-heralded method within biodiversity science.The first true eDNA studies in the field date back only fifteen years<a href="#fn9" id="fnef9" role="doc-noteref" data-turbo="false"><sup>9</sup></a>. In that time, thousands of studies have been published using eDNA data, sampled from everywhere from tide pools and marshlands to uranium mine containment ponds and cargo ship ballast tanks. While many eDNA studies operate through traditional professional research teams, the ease of sampling eDNA has contributed to the popularity of so-called citizen science projects using it as a tool.&nbsp;</p><p style="text-align:start">Citizen science — a term first used to describe UFO-spotting projects in the 1970s<a href="#fn10" id="fnef10" role="doc-noteref" data-turbo="false"><sup>10</sup></a> — is a type of scientific research where some portion of the methods are performed not by professional scientists but volunteers: hobbyists who collect samples, tag data, or contribute computing power to aid large projects.&nbsp;</p><p style="text-align:start">The citizen science project is a hierarchical, or top-down, enterprise: while its actors are a broad, distributed group of non-experts, its leaders both in terms of ideation and coordination are the academics and scientists who conduct traditional studies regardless. A slightly different term — <em>community science </em>— is used to refer to projects with a more horizontal or bottom-up configuration, wherein research questions and methodologies are built with community input.</p><p style="text-align:start">Regardless of their precise form, eDNA projects have made effective use of mass public sampling over the course of the last decade. In Denmark in 2019 and 2020, a coordinated, mass sampling event, or an eDNA “bioblitz,” collected four hundred samples in total from one hundred locations, a geographic spread that would have been prohibitively expensive for a traditional survey<a href="#fn11" id="fnef11" role="doc-noteref" data-turbo="false"><sup>11</sup></a>.</p><p style="text-align:start">Unlike traditional biodiversity surveying, eDNA sample collection requires no underlying familiarity with the ecological communities of the region. The ideal eDNA citizen science participant is not the seasoned trail veteran venerated in many well-cited articles about the potential of citizen science<a href="#fn12" id="fnef12" role="doc-noteref" data-turbo="false"><sup>12</sup></a>. In fact, some studies have pointed to eDNA as a way to accommodate for “a continuous decline in taxonomic expertise” within the scientific community<a href="#fn13" id="fnef13" role="doc-noteref" data-turbo="false"><sup>13</sup></a>. An eDNA community science participant need not have any skill at identifying bird calls or ungulate tracks. They simply need to be willing to collect bottles of water or bags of sediment with an abundance of patience.</p><p style="text-align:start">In a sense, this lack of required background knowledge makes eDNA sampling a welcoming entry point for a broader public — people who appreciate the natural world without yet having a rigorous scientific understanding of it. Dr. Kim Ballare is an ecologist at the University of California, Santa Cruz and a leader in CALeDNA, a research-focused eDNA initiative. Dr. Ballare has found that eDNA studies are useful for introducing interested newcomers to ecology. In her view, eDNA community science projects help bring in new participants to the scientific process by bypassing the “tedious and intimidating” aspects of traditional methods like transect sampling and providing easier access to data for researchers without extensive field experience. In her work with undergraduate students, Dr. Ballare has used eDNA not just as a research tool but also a teaching tool — a way of easily uncovering hidden layers of biodiversity, like the rich universes of fungal and microbial taxa, that are difficult to tease out through conventional methods.&nbsp;</p><h3 style="text-align:start" id="the-fruits-of-scientific-labor">The Fruits of Scientific Labor</h3><p style="text-align:start">The low barrier of entry to participating in eDNA research is one of its greatest strengths, but it also reveals a gap in the technology’s promise. The eDNA volunteer is not required to know very much about ecology before they participate, but they don’t always learn much from the process, either. The eDNA volunteer is left without a clear path to become more invested in the ultimate results of the projects they are a part of, or to begin new scientific projects of their own.&nbsp;</p><p style="text-align:start">Unlike even simple traditional community science exercises — Christmas bird counts, ant censuses, and the like — eDNA citizen science projects do not necessarily involve seeing even a single distinguishable organism. The hundreds or thousands of species that leave their mark in a sample are obscured to its collector. As Dr. Ross Mounce, a evolutionary biologist and open science advocate currently working as Director of Open Access Programmes at Arcadia Fund notes, eDNA sampling leaves out “All the interesting things about a species: how it moves, how it reproduces, how it defecates, does it fart” — precisely the things that “citizen science scientists should be exposed to and should learn in the process of doing systems science.”</p><p style="text-align:start">While the act of sample collection itself has a low barrier to entry, the fruits of that scientific labor are often not redistributed back to those who produced its raw material. Instead, the steps in eDNA studies following sample collection and filtration often inadvertently make scientific results more obscure to research participants. If the research project a volunteer contributed to isn’t connected to a lab that can perform eDNA sequencing, it can take additional weeks or even months to process samples. The structures of academic publishing make it all the less likely that the results make it out into the broader world in a timely fashion, if they ever do. While the scientists leading studies can afford to stay with them until they reach fruition — it is literally their job to do so — the volunteers who participate in a single bioblitz or sampling expedition can frequently fall out of the loop.</p><h3 style="text-align:start" id="semi-permeable-science">Semi-Permeable Science</h3><p style="text-align:start">Environmental DNA’s revolutionary potential is undermined by its inaccessibility as a scientific tool. As with many technologies with much hyped transformative potentials attached to them — in the world of biology and beyond — eDNA’s reach is not evenly distributed yet.&nbsp;</p><p style="text-align:start">On a basic level, this inaccessibility comes from material and technological constraints. As Dr. Ballare notes, “not every institution has the ability to process these [samples], you need very specific technology and it can be pretty expensive.” The cost of processing DNA samples has decreased significantly over the past two decades — one benchmark, the cost to sequence a single human genome, has fallen from $10,000 to $200 over the last ten years<a href="#fn14" id="fnef14" role="doc-noteref" data-turbo="false"><sup>14</sup></a>. As these costs decrease, the hope is that eDNA methods can become feasible to use in a wider set of circumstances and by a broader group of researchers.</p><p style="text-align:start">Yet the barriers to accessing eDNA data go beyond technical and financial limitations. Studies using eDNA metabarcoding already do well on certain basic measures of open science — they are largely published in open access journals, and their supporting genetic data is typically stored in centralized, publicly available repositories like the NCBI’s Sequence Read Archive<a href="#fn15" id="fnef15" role="doc-noteref" data-turbo="false"><sup>15</sup></a>.&nbsp;</p><p style="text-align:start">In practice, these attempts at data accessibility rarely go beyond surface-level implementations of open science ideals. One review paper published in 2021 found that only two out of the forty-four relevant articles published in the journal <em>Environmental DNA </em>provided their data on platforms where the data would be “formatted for simple re-use” or made usable alongside other genetic data<a href="#fn16" id="fnef16" role="doc-noteref" data-turbo="false"><sup>16</sup></a>. While open access-focused organizations like GBIF, the Global Biodiversity Information Facility, have more recently proposed standards for the publishing and interoperability of eDNA data, it’s unclear how much uptake these standards will have within the broader scientific community<a href="#fn17" id="fnef17" role="doc-noteref" data-turbo="false"><sup>17</sup></a>.</p><p style="text-align:start">CALeDNA provides perhaps the most promising set of best practices for eDNA community science studies. Since the project’s inception in 2017, CALeDNA has prioritized data accessibility. Data collected from its community science studies is made available on a public portal, with results searchable by location and species sampled<a href="#fn18" id="fnef18" role="doc-noteref" data-turbo="false"><sup>18</sup></a>. CALeDNA’s team is also at work on eDNA Explorer, a full-fledged site for sharing eDNA data and techniques with the aim, as Ballare notes, “to foster collaboration with scientists and with the public”<a href="#fn19" id="fnef19" role="doc-noteref" data-turbo="false"><sup>19</sup></a>. CALeDNA studies also provide community science volunteers with a fairly large degree of methodological leeway — their FAQ advertises that “You get to choose!” the experimental design<a href="#fn20" id="fnef20" role="doc-noteref" data-turbo="false"><sup>20</sup></a>.&nbsp;</p><p style="text-align:start">As Dr. Ballare admits, though, “it’s mostly the collection still that the community is involved in.” Later stages of community science eDNA projects are still largely the province of formally-trained scientists, in large part due to the sensitivity of laboratory materials and filtered samples. Yet even after DNA has been extracted, sequenced, and analyzed, community science participants are often left out of the loop. CALeDNA reaches out to its volunteers through a regular newsletter with updates on its projects, but does not always contact volunteers about the particular projects that they’re involved with. Other projects are less clear about their outreach efforts — one-off initiatives tend to have weaker documentation of their process.</p><h3 style="text-align:start" id="edna-for-the-people">eDNA for the People</h3><p style="text-align:start">The nuances surrounding eDNA community science studies tie deeply to an ongoing debate regarding open access to science and the ethics of volunteer data in science. Platforms like iNaturalist, which draw in millions of community science observations and ingest them into repositories like GBIF, have inspired scientific debate about how to cite individual contributions within a vast corpus. Dr. Mounce, the open science expert, holds that proper citation of community science participants is key to ethical science, but notes with dismay that&nbsp; “oftentimes journals aren’t very good at policing” citation practices for nontraditional sources.</p><p style="text-align:start">Yet, Dr. Mounce says, there are many “really exciting” initiatives in the world of open science broadly. The Open Climate Campaign, a project started by Creative Commons, SPARC, and EIFL, aims to focus the open access movement’s energy on climate change and the biodiversity crisis. Their ultimate goal is to “promote open access to research to accelerate progress towards solving the climate crisis and preserving global biodiversity”<a href="#fn21" id="fnef21" role="doc-noteref" data-turbo="false"><sup>21</sup></a>. Democratizing access to eDNA data would be a key step towards that goal.</p><p style="text-align:start">As it currently stands, marine environmental DNA research is still largely the province of a rarefied set of researchers, mostly clustered in elite academic institutions in North America, Western Europe, and the Pacific Rim.<a href="#fn22" id="fnef22" role="doc-noteref" data-turbo="false"><sup>22</sup></a> When eDNA research projects do collect samples from areas outside of those core regions, the research is still ultimately led by teams originating within them, without significant input from the people who live in or near sampling locations. The economic realities of the current regime of eDNA research promote what certain commentators refer to as “Parachute Science” — well-funded, Western researchers dropping into impoverished sites, collecting their samples without attention to local context, and reaping the scientific benefits<a href="#fn23" id="fnef23" role="doc-noteref" data-turbo="false"><sup>23</sup></a>.</p><h3 style="text-align:start" id="back-to-the-water">Back to the Water</h3><p style="text-align:start">Environmental DNA analysis is not a cure-all to the complex set of challenges facing the contemporary ecologist. It’s one tool out of many, and an imperfect one at that. eDNA is best used as one piece within a broader puzzle of biodiversity monitoring, coupled with traditional modes of sampling. Yet its appeal is not in reducing the rich tapestry of life down to mere sequences but in the reverse: its ability to open the doors of the natural world to the people who seek to understand it, regardless of scientific background or expertise. For now, at least, it does not quite capture that theoretical appeal in full.</p><p style="text-align:start">There are technical and procedural changes that can help get us there — norms and infrastructures that promote open data access and transparency throughout the scientific process. Initiatives like the CALeDNA Explorer and other open access projects within eDNA are laying the groundwork for a future where creating and accessing eDNA data is not some rarefied scientific act, but a commonplace part of being in and understanding nature.&nbsp;</p><p style="text-align:start">Technical fixes alone will not turn eDNA into a tool of scientific liberation, though. The barriers erected around scientific data are far older than eDNA, and the possessive logic of ancient kings and Victorian naturalists still has its echoes in every paywalled paper and unavailable dataset. Even so, the very act of eDNA sampling — of transforming water and your own patience into scientific data — provides a counterbalance of hope.&nbsp;</p><p style="text-align:start">In the moment, with the currents rushing through the tide pool and your hands rushing through the water, the hidden patterns of the ecological world seem to flow all around you. As you bring life out of the water, it feels impossible to be cynical about eDNA’s potential.</p><ol>
<li id="fn1">Smith-Root, Inc. “EDNA SAMPLER BACKPACK COMBO,” n.d. https://store.smith-root.com/products/edna-sampler-backpack-lith-combo <a href="#fnef1" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn2">Oka, Shinichiro, Masaki Miya, and Tetsuya Sado. “Gravity Filtration of Environmental DNA: A Simple, Fast, and Power-Free Method.” <em>MethodsX</em> 9 (August 1, 2022): 101838. https://doi.org/10.1016/j.mex.2022.101838 <a href="#fnef2" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn3">Harvard University. “Museum of Comparative Zoology,” n.d. https://mcz.harvard.edu/. <a href="#fnef3" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn4">Richards, Thomas. <em>The Imperial Archive: Knowledge and the Fantasy of Empire</em>. Verso, 1993. <a href="#fnef4" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn5">Loh, Rebecca Ker, Sujatha Narayanan Kutty, Darren C. J. Yeo, and Rudolf Meier. “Toward EDNA‐based Bioassessment of Freshwater Reservoirs with Small Volumes of Water: Robust Molecular Protocols.” <em>Environmental DNA</em>, December 7, 2022. https://doi.org/10.1002/edn3.373. <a href="#fnef5" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn6">ScienceDaily. “The Ocean in a Cup: Environmental DNA Successfully Captures Marine Biodiversity,” November 22, 2022. https://www.sciencedaily.com/releases/2022/11/221116164919.htm. <a href="#fnef6" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn7">Kjær, Kurt H., Winther Pedersen Mikkel, De Sanctis Bianca, De Cahsan Binia, Thorfinn S. Korneliussen, Christian S. Michelsen, Karina K. Sand, et al. “A 2-Million-Year-Old Ecosystem in Greenland Uncovered by Environmental DNA.” <em>Nature</em> 612, no. 7939 (December 7, 2022): 283–91. https://doi.org/10.1038/s41586-022-05453-y <a href="#fnef7" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn8">Saito, Tatsuya, and Hideyuki Doi. “Degradation Modeling of Water Environmental DNA: Experiments on Multiple DNA Sources in Pond and Seawater.” <em>Environmental DNA</em> 3, no. 4 (March 25, 2021): 850–60. https://doi.org/10.1002/edn3.192 <a href="#fnef8" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn9">Ficetola, Gentile Francesco, Claude Miaud, François Pompanon, and Pierre Taberlet. “Species Detection Using Environmental DNA from Water Samples.” <em>Biology Letters</em> 4, no. 4 (August 23, 2008): 423–25. https://doi.org/10.1098/rsbl.2008.0118. <a href="#fnef9" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn10">Oberg, James. “The Failure of the ‘science’ of UFOlogy.” <em>New Scientist</em>, October 11, 1979. <a href="#fnef10" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn11">Agersnap, Sune, Eva Egelyng Sigsgaard, Mads Christian Jensen, Marcelo De Paula Avila, Henrik Carl, Peter Møller, Simon Leed Krøs, Steen Knudsen, Mary S. Wisz, and Philip Francis Thomsen. “A National Scale ‘BioBlitz’ Using Citizen Science and EDNA Metabarcoding for Monitoring Coastal Marine Fish.” <em>DOAJ (DOAJ: Directory of Open Access Journals)</em>, March 28, 2022. https://doi.org/10.3389/fmars.2022.824100. <a href="#fnef11" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn12">Cohn, Jeffrey F. “Citizen Science: Can Volunteers Do Real Research?” <em>BioScience</em> 58, no. 3 (March 1, 2008): 192–97. https://doi.org/10.1641/b580303. <a href="#fnef12" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn13">Thomsen, Philip Francis, and Eske Willerslev. “Environmental DNA – An Emerging Tool in Conservation for Monitoring Past and Present Biodiversity.” <em>Biological Conservation</em> 183 (March 1, 2015): 4–18. https://doi.org/10.1016/j.biocon.2014.11.019. <a href="#fnef13" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn14">Mullin, Emily. “The Era of Fast, Cheap Genome Sequencing Is Here.” <em>WIRED</em>. September 29, 2022. https://www.wired.com/story/the-era-of-fast-cheap-genome-sequencing-is-here/. <a href="#fnef14" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn15">“Home - SRA - NCBI,” n.d. https://www.ncbi.nlm.nih.gov/sra. <a href="#fnef15" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn16">Berry, Oliver, Simon N. Jarman, Andrew Bissett, Michael D. Hope, C. Paeper, Cindy Bessey, Michael W. Schwartz, Josh Hale, and Michael Bunce. “Making Environmental DNA (EDNA) Biodiversity Records Globally Accessible.” <em>Environmental DNA</em> 3, no. 4 (July 1, 2021): 699–705. https://doi.org/10.1002/edn3.173. <a href="#fnef16" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn17">Finstad, Anders Gravbrøt, Anders Andersson, Andrew Bissett, Frode Fossøy, Marie Grosjean, Michael Hope, Urmas Kõljalg, et al. 2021. “Publishing Sequence-Derived Data through Biodiversity Data Platforms.” <em>GBIF</em>. <a href="https://doi.org/10.35035/DOC-VF1A-NR22" rel="noopener" target="_blank" referrerpolicy="strict-origin-when-cross-origin">https://doi.org/10.35035/DOC-VF1A-NR22</a>. <a href="#fnef17" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn18">CALeDNA. “Explore Data,” n.d. https://data.ucedna.com/samples <a href="#fnef18" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn19">&nbsp;“EDNA Explorer - Coming Soon,” n.d. https://www.ednaexplorer.org/. <a href="#fnef19" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn20">CALeDNA. “FAQ — CALeDNA,” n.d. https://ucedna.com/frequently-asked-questions. <a href="#fnef20" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn21">Open Climate Campaign. “Home – Open Climate Campaign.” Open Climate Campaign – a 4-year Campaign to Make Open Sharing of Research Outputs the Norm in Climate Science., November 1, 2022. https://openclimatecampaign.org/. <a href="#fnef21" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn22">Shea MM, Kuppermann J, Rogers MP, Smith DS, Edwards P, Boehm AB.&nbsp;2023.&nbsp;Systematic review of marine environmental DNA metabarcoding studies: toward best practices for data usability and accessibility.&nbsp;<em>PeerJ</em>&nbsp;11:e14993 https://doi.org/10.7717/peerj.14993 <a href="#fnef22" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
<li id="fn23">Barber, Paul H., Ma. Carmen Ablan-Lagman, Ambariyanto, Roberto G. S. Berlinck, Dita J. Cahyani, Eric D. Crandall, Rachel Ravago-Gotanco, et al. “Advancing Biodiversity Research in Developing Countries: The Need for Changing Paradigms.” <em>Bulletin of Marine Science</em> 90, no. 1 (January 1, 2014): 187–210 <a href="#fnef23" role="doc-backlink" data-turbo="false">↩︎︎</a>
</li>
</ol>